docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -107,20 +75,35 @@ spec:
       vlan: 22
 EOF
 ----
++
+* In case when `neutron-sriov-nic-agent` is running on the existing Compute nodes, physical device mappings needs to be checked and set the same in the
+`OpenStackDataPlaneNodeSet` custom resource (CR). Those options will need to be set in the `OpenStackDataPlaneNodeSet` CR. For more information, see xref:reviewing-the-openstack-control-plane-configuration_adopt-control-plane[Reviewing the {rhos_prev_long} control plane configuration].
+
+.Variables
+
+Define the shell variables used in the Fast-forward upgrade steps below.
+Set `FIP` to the floating IP address of the `test` VM pre-created earlier on the source cloud.
+Define the map of Compute node name, IP pairs.
+The values are just illustrative, use values that are correct for your environment:
+
+----
+PODIFIED_DB_ROOT_PASSWORD=$(oc get -o json secret/osp-secret | jq -r .data.DbRootPassword | base64 -d)
 
-* In case when neutron-sriov-nic-agent is running on the existing compute nodes,
-  physical device mappings needs to be checked and set the same in the
-  OpenStackDataPlaneNodeSet CR. Those options will need to be set in the
-  OpenStackDataPlaneNodeSet CR.
-  For more information, see xref:pulling-the-openstack-configuration_{context}[Pulling the OpenStack configuration].
+alias openstack="oc exec -t openstackclient -- openstack"
+FIP=192.168.122.20
+declare -A computes
+export computes=(
+  ["standalone.localdomain"]="192.168.122.100"
+  # ...
+)
+----
 
-== Adopting the data plane
+.Procedure
 
 * _Temporary fix_ until the OSP 17 https://code.engineering.redhat.com/gerrit/q/topic:stable-compute-uuid[backport of the stable compute UUID feature]
```

Comment:
Consider changing this from an unordered list to an admonition of some type? Note? Important?

Reply:
Yes, but we might be removing it after 17.1.3 lands. I have a comment in the file to revisit this.